### PR TITLE
Cleaned up warnings and provided some error checking in build.php

### PR DESCRIPTION
**Pull request by deanhouseholder**

### CODE REVIEW

File: build.php [hunk 1]
```diff
@@ -26,32 +26,45 @@
 use Seld\PharUtils\Timestamps;
 use Symfony\Component\Finder\Finder;
 
-require_once __DIR__.'/vendor/autoload.php';
+$dir = __DIR__;
```
Comment (jnvsor) [inline build.php]: Could you put this back? I don't like the idea of making a variable for what's a constant anyway, and I doubt the build folder will ever change

File: build.php [hunk 2]
```diff
@@ -26,32 +26,45 @@
...
+$build_dir = $dir.'/build';
+$outpath = $build_dir.'/kint.phar';
 
-\mkdir(__DIR__.'/build');
+require_once $dir.'/vendor/autoload.php';
 
-$outpath = __DIR__.'/build/kint.phar';
+if (ini_get('phar.readonly') == 1) {
+    die("Error: Creating phar files is disabled by your system.\nYou must set \"phar.readonly = Off\" in your php.ini to continue.\n");
```
Comment (jnvsor) [inline build.php]: I think the PHP builtin error message is clear enough, and will get more search hits if people google it:

> PHP Fatal error:  Uncaught UnexpectedValueException: creating archive ".../kint/build/kint.phar" disabled by the php.ini setting phar.readonly in .../kint/build.php:36

**Review by jnvsor — CHANGES_REQUESTED** [review]
Appreciate the file_exists checks though (And remember to run it through format or CI will complain!)

**Comment by deanhouseholder** [comment]
I'm going to cancel and submit a new one.

**Pull request closed without merging**